client/my-sites/stats/stats-summary/index.jsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import { isEqual, find } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import ElementChart from 'components/chart';
+import StatsTabs from '../stats-tabs';
+import StatsTab from '../stats-tabs/tab';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import Card from 'components/card';
+import { recordGoogleEvent } from 'state/analytics/actions';
+
+class StatsSummaryChart extends Component {
+	static propTypes = {
+		data: PropTypes.array,
+		dataKey: PropTypes.string.isRequired,
+		isLoading: PropTypes.bool,
+		labelClass: PropTypes.string.isRequired,
+		labelKey: PropTypes.string.isRequired,
+		sectionClass: PropTypes.string.isRequired,
+		selected: PropTypes.object,
+		tabLabel: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		onClick: () => {}
+	};
+
+	barClick = bar => {
+		const selectedBar = find( this.props.data, data => isEqual( data, bar.data ) );
+		this.props.recordGoogleEvent( 'Stats', 'Clicked Summary Chart Bar' );
+		this.props.onClick( selectedBar );
+	};
+
+	buildChartData() {
+		const { data, labelClass, sectionClass, selected, tabLabel } = this.props;
+		return data.map( record => {
+			const className = classNames( {
+				'is-selected': isEqual( selected, record )
+			} );
+
+			const tooltipData = [
+				{
+					label: record.periodLabel || record.period,
+					className: 'is-date-label',
+					value: null
+				},
+				{
+					label: tabLabel,
+					className: sectionClass,
+					value: record.value,
```

Comment:
Should we use `[numberFormat](https://github.com/Automattic/i18n-calypso#numberformat-method)` here so large numbers are properly displayed/localized?